server_test.go

```diff
@@ -192,10 +192,9 @@ func BenchmarkChainUnaryInterceptor(b *testing.B) {
 			s := NewServer(ChainUnaryInterceptor(interceptors...))
 			b.ReportAllocs()
 			b.ResetTimer()
+			ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
+			defer cancel()
```

Comment:
This is still inside a loop.  And actually moving it out would be important here, since it would impact the benchmark.

Reply:
Sure, LMK if you meant something different.